src/main/java/com/online/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,11 +1,18 @@
 package com.online.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
 public record CreateBookRequestDto(
+        @NotNull
         String title,
+        @NotNull
         String author,
+        @NotNull
         String isbn,
+        @NotNull
+        @Min(0)
```

Comment:
Let's validate more detail, like Not Blank, ISBN anno, limit or smth else

Reply:
validated